### DISCUSSION TITLE
ZPL failing in Virtual-ZPL-Printer

### DISCUSSION THREAD

**Discussion by WettDawg** [porrey/Virtual-ZPL-Printer#6] [Q&A] [answered]
Hi, I have just installed this and it is working like a champ!  Great utility!!  However, I am getting an error with one label type.  It works in the Labelary online decoder, but, it is failing here.  The error is "You requested the 1st label, but the specified ZPL doesn't generate any labels".  I do get many warnings about the values being too large or less than a minimum value and it ignores or truncates (see below).  Could these be causing it not to work in the app?  Any help would be greatly appreciated!  Thanks

[^FT: Value 1222 is greater than label height 1216; field will not be visible on the label]
[^FT: Value 1230 is greater than label height 1216; field will not be visible on the label]
[^BY: Value 4 is greater than maximum value 3; used 3 instead]
[^BY: Value 16 is greater than maximum value 3; used 3 instead]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BY: Value 16 is greater than maximum value 3; used 3 instead]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BY: Value 4 is greater than maximum value 3; used 3 instead]
[^A: Value 'N21' is too long for an orientation (N/R/I/B) and was trimmed to one character]
[^FT: Value 1278 is greater than label height 1216; field will not be visible on the label]
[^FT: Value 1277 is greater than label height 1216; field will not be visible on the label]
[^BY: Value 54 is greater than maximum value 3; used 3 instead]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BY: Value 54 is greater than maximum value 3; used 3 instead]
[^BX: Value 0 is less than minimum value 1 and was ignored]
[^BX: Value 0 is less than minimum value 1 and was ignored]

**Comment by porrey** [comment]
Are you able to share the ZPL or send it to me so I can debug it?

**Reply by WettDawg** [reply]
I would be happy to send it to you directly.  I have obfuscated some info, but, would like to NOT post it here.

**Comment by porrey** [comment, marked as answer]
I found an issue with the receive buffer which results in the label being truncated (greater than 8,192 bytes). A fix has been posted.

**Reply by WettDawg** [reply]
That's fixed it :)  Thank you!!
